routers/test.ts

```diff
@@ -0,0 +1,14 @@
+import express from 'express';
+
+import testController from '../controllers/testController';
+
+import config from '../utils/config';
+
+const router = express.Router();
+
+if (config.testEnvironment) {
```

Comment:
Maybe better to put this around `router.use('/test', testRouter);` in `routers/index.ts`. (not blocking PR approval)